src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -464,6 +421,95 @@ Sentry.withScope(function(scope) {
 });
 ```
 
+### Extra Context {#extra-context}
+In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening:
+
+```javascript
+Sentry.setExtra("character_name", "Mighty Fighter");
+```
```

Comment:
We decided to de-emphasize the use of guide users to call `setExtra` we always want them to use `setContext`

I know this was there before but let's fix it now

Reply:
Fixed